Android.gitignore

```diff
@@ -1,89 +1,31 @@
-# Built application files
-*.apk
-*.aar
-*.ap_
-*.aab
-
-# Files for the ART/Dalvik VM
-*.dex
-
-# Java class files
-*.class
-
-# Generated files
-bin/
-gen/
-out/
-#  Uncomment the following line in case you need and you don't have the release build type files in your app
-# release/
-
 # Gradle files
 .gradle/
 build/
 
 # Local configuration file (sdk path, etc)
 local.properties
 
-# Proguard folder generated by Eclipse
-proguard/
-
-# Log Files
+# Log/OS Files
 *.log
+.DS_Store
```

Comment:
```suggestion
```
I am going to remove this change since it is still unresolved. My recommendation is still to use the Global Macos file for this.